vendor/bat-native-ads/src/bat/ads/internal/purchase_intent/funnel_sites_unittest.cc

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <stdint.h>
+#include <memory>
+#include <tuple>
+
+#include "bat/ads/internal/ads_client_mock.h"
+#include "bat/ads/internal/ads_impl.h"
+
+#include "testing/gtest/include/gtest/gtest.h"
+
+#include "bat/ads/internal/purchase_intent/funnel_sites.h"
+
+using ::testing::_;
+
+// npm run test -- brave_unit_tests --filter=AdsPurchaseIntentFunnelSites*
+
+namespace {
+
+struct TestTriplet {
+  std::string url;
+  ads::FunnelSiteInfo funnel_site_info;
+};
+
+const std::vector<TestTriplet> kTestUrls = {
+  {"http://www.carmax.com", ads::_automotive_funnel_sites.at(1)},
+  {"http://www.carmax.com/foobar", ads::_automotive_funnel_sites.at(1)},
+  {"http://carmax.com", ads::_automotive_funnel_sites.at(1)},
+  {"http://brave.com/foobar", ads::FunnelSiteInfo()},
+};
+
+}  // namespace
```

Comment:
Can we please move this inside the `ads` namespace and remove `ads::` from these